src/main/java/firebase/sdk/dashboard/api/UserResource.java

```diff
@@ -0,0 +1,69 @@
+package firebase.sdk.dashboard.api;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import firebase.sdk.dashboard.dao.PlatformReleaseDao;
+import firebase.sdk.dashboard.dao.PlatformReleaseDaoDatastore;
+import firebase.sdk.dashboard.dao.SDKDao;
+
+/**
+ * Root resource (exposed at "myresource" path)
+ */
+@Path("/users")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class UserResource {
+
+  private ArrayList<String> favourites;
+  public UserResource() {
+    favourites = new ArrayList<String>(Arrays.asList("firebase-common", "firebase-common-ktx", "firebase-installations", "firebase-installations-interop"));
+  }
+
+  //private static final PlatformReleaseDao PRDAO = new PlatformReleaseDaoDatastore();
```

Comment:
Delete this comments if not required else add a TODO on when you plan tp use this

Reply:
I still see the commented code here. Maybe an issue with viewing the latest changes. Please double check.